tags/tags.toml

```
@@ -762,10 +762,17 @@ keywords = ["embed-files", "embed files"]
 content = """
 `MessageEmbed#attachFiles` has been removed. Files should be attached via the message option object instead:
 ```diff
-- const embed = new MessageEmbed().setTitle('Attachments').attachFiles(['./image1.png', './image2.jpg']);
+  const attachment = new MessageAttachment('./image.png', 'image1.png');
+  const embed = new MessageEmbed()
+    .setTitle('Attachments')
+    .setImage(`attachment://${attachment.name}`)
+-   .attachFiles([attachment]);
```

We could do it like this:
```suggestion
-   .attachFiles([attachment])
    .setTitle('Attachments')
    .setImage(`attachment://${attachment.name}`);
```

With the current code, the removal of the line would result on the removal of the semicolon, and without the rearrangement, the diff would be messier:

```diff
-   .setImage(`attachment://${attachment.name}`)
-   .attachFiles([attachment]);
+   .setImage(`attachment://${attachment.name}`);
```